### ISSUE TITLE
Custom window layouts or staggered windows

### ISSUE THREAD

**Issue by mkbt** [fikovnik/ShiftIt#182]
So, I tend to lay my windows out in various cascaded ways and jump between them. It would be nice if there was a way I could template my favourite window layouts and assign them to a key command.

See attached image for what  I mean.

Is this crazy?

![screen shot 2015-08-06 at 11 50 21 am 2](https://cloud.githubusercontent.com/assets/550077/9102053/5ef8cc46-3c31-11e5-8747-dfb6cbe6c7e6.png)

**Comment by fikovnik** [comment]
No, this is a very valid request. I have already planned to rework the code base of ShiftIt so it can be easily added.

**Comment by miqmago** [comment]
Thanks for ShiftIt. This featyre would be really helpfull.
Currently, before start working in a workspace I'm organizing the windows in the following way and this feature would speed up this process:

For each window:

- ^+Alt+Cmd+m
- ^+Alt+Cmd+'
- Place the window in cascade

**Comment by vladimirprieto** [comment]
so what i understood of what @fikovnik said is there will be some kind of layouts schema store.  right?

i want to know cuz to avoid to creat a feature request for that.

thanks.

**Comment by fikovnik** [comment]
I haven't plan to work on this anytime soon so feel free.
